website/source/docs/install/index.html.md

```diff
@@ -50,7 +50,7 @@ a copy of [`git`](https://www.git-scm.com/) in your `PATH`.
   needed to compile Consul:
 
     ```shell
-    $ make bootstrap
+    $ make
```

Comment:
Thanks for taking a look at this! I think we now want it to be:

```
$ make tools
```

I believe `make` without arguments will also build binaries for all platforms, which isn't what we want in this guide.

Reply:
I see that in the makefile now. I'll make the correction. Thanks for pointing that out!